core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -53,6 +54,32 @@ oppia.controller('ExplorationHistory', [
     }
   });
 
+  $scope.submitVersion = function($event, versionNumber) {
+    var checkbox = $event.target;
+    var id = $scope.versionArr.indexOf(versionNumber);
+    if (checkbox.checked && id === -1) {
+      $scope.versionArr.push(versionNumber);
+    }
+    if (!checkbox.checked && id !== -1) {
+      $scope.versionArr.splice(id, 1);
+    }
+    
+    if($scope.versionArr.length == 2) {
+      $scope.versionCountPrompt = "";
+      $scope.changeCompareVersion(versionNumber);
+    }
+    else {
+      $scope.hideHistoryGraph = true;
+      $scope.hideCompareVersionsButton = false;
+
+      if($scope.versionArr.length == 0)
+        $scope.versionCountPrompt = "Please select any two.";
+      else if($scope.versionArr.length == 1)
+        $scope.versionCountPrompt = "Please select one more.";
+      else 
+        $scope.versionCountPrompt = "Please select exactly two.";      
```

Comment:
This seems to differ from the mock that @ben-targan gave. Ben's mock had checkboxes disabled after 2 were checked, while this prompts the user to select less. Did you discuss this with @ben-targan?